drivers/modem/hl78xx/Kconfig.hl78xx

```
@@ -0,0 +1,422 @@
+# Sierra Wireless HL78XX driver driver options
+
+# Copyright (c) 2025 Netfeasa
+# SPDX-License-Identifier: Apache-2.0
+
+config MODEM_HL78XX
+	bool "HL78XX modem driver"
+	select MODEM_MODULES
+	select MODEM_CHAT
+	select MODEM_PIPE
+	select MODEM_PIPELINK
+	select MODEM_BACKEND_UART
+	select RING_BUFFER
+	select MODEM_SOCKET
+	select NET_OFFLOAD
+	select MODEM_CONTEXT
+	select EXPERIMENTAL
+	depends on !MODEM_CELLULAR
+	imply GPIO
+	help
+		Choose this setting to enable Sierra Wireless HL78XX driver LTE-CatM1/NB-IoT modem
+		driver.
```

Help indentation should be two extra spaces, not tab:
```suggestion
	help
	  Choose this setting to enable Sierra Wireless HL78XX driver LTE-CatM1/NB-IoT modem
	  driver.
```
Same thing elsewhere in the file.

---

done